config/nuxt/modules/nuxt-robots.js

```diff
@@ -0,0 +1,8 @@
+import env from '../env'
+
```

Comment:
link to module docs like you did for `nuxt-sitemap` so a next developer can easily find it